index.js

```diff
@@ -59,14 +62,12 @@ async function create ({
       })
     }
 
-    let hasUploaded = false
-
-    metadata.on('upload', () => {
-      hasUploaded = true
-    })
-
     await once(metadata, 'peer-open')
 
+    const stats = await updateStats(drive)
```

Comment:
Would `getFileRanges` or something like that make sense for the name? `uploadStats` sounds like it's uploading stats on the network somewhere.

Reply:
yeah, lets go with `getFileRanges`. Naming is hard 😭